### DISCUSSION TITLE
How can I access the sidekiq and pghero web UI?

### DISCUSSION THREAD

**Discussion by felixriehm** [mastodon/mastodon#22614] [Q&A] [answered]
When installing Mastodon on a remote server without docker, how can I access the sidekiq and pghero web UI that was present in Mastodon v3? I can remember that in v3 there were two links in the admin panel but not anymore. In my case just trying "https://mydomain/sidekiq/" returns 404.

**Comment by alexandru** [comment, marked as answer]
Accessing the `/sidekiq/` page required the `Owner` role on my instance. `Admin` wasn't enough, and I was getting 404, just like you.

**Reply by felixriehm** [reply]
Thanks! I just checked. You are right 👍
